docs/source/tutorial/basic.rst

```diff
@@ -250,16 +250,16 @@ Now you can solve a multiclass classification task using a multi-layer perceptro
 Here we use hand-written digits dataset called `MNIST <http://yann.lecun.com/exdb/mnist/>`_, which is the long-standing de-facto "hello world" of machine learning.
 This MNIST example is also found in ``examples/mnist`` directory of the official repository.
 
-In order to use MNIST, :func:`sklearn.datasets.fetch_mldata` function of `scikit-learn <http://scikit-learn.org/>`_ is useful::
+In order to use MNIST, we prepared :func:`data.load_mnist_data` function::
```

Comment:
The `data` module is defined in the MNIST example, so
- It cannot be referred by `:func:`.
- I think we should add a note it here that the definition is at the `examples/mnist/data.py`.